services/checks/checks.go

```diff
@@ -594,6 +627,11 @@ func (s *Service) minPMMAgentVersion(t check.Type) *version.Parsed {
 	case check.MongoDBGetDiagnosticData:
 		return pmmAgent2_27_0
 
+	case check.MetricsRange:
+		fallthrough
+	case check.MetricsInstant:
+		return pmmAgentAny
```

Comment:
looks like returning nil will be ok and work faster

Reply:
Yep, done